src/main/scala/coreplex/BaseCoreplex.scala

```diff
@@ -32,7 +32,7 @@ trait HasCoreplexParameters {
   lazy val lsb = p(BankIdLSB)
   lazy val innerParams = p.alterPartial({ case TLId => "L1toL2" })
   lazy val outermostParams = p.alterPartial({ case TLId => "Outermost" })
-  lazy val outermostMMIOParams = p.alterPartial({ case TLId => "MMIO_Outermost" })
+  lazy val outerMMIOParams = p.alterPartial({ case TLId => "L2toMMIO" })
```

Comment:
It really doesn't matter (with TL2 rollout), but I think innerMMIOParams is better than outerMMIOParams.  IMO, it's more confusing to have outerMMIOParams vs. outermostMMIOParams.